### ISSUE TITLE
Tablet Landscape vs Portrait mode

### ISSUE THREAD

**Issue by alfureu** [FreshRSS/FreshRSS#2187]
The problem I am having is with the inconsistency between landscape and portrait mode while using FreshRSS on a **tablet**. I use the following extensions:

- [x] CustomCSS | 0.2 
- [x] CustomJS | 0.2 
- [x] ImageProxy | 0.4 
- [x] StickyFeeds | 0.1 
- [x] YouTube | 0.7 
- [x] Dilbert | 0.3 
- [x] Teem | 0.1 
- [x] FixedNavMenu | 0.1 
- [x] KeepFolderState | 0.1 
- [x] MobileScrollMenu | 0.2 
- [x] SmartMobileMenu | 0.1 
- [x] TouchControl | 0.2 
- [x] AutoRefresh | 1 |
- [x] RedditImage | 0.1 
- [x] ReadingTime | 1

On a Tablet in portrait-mode the left side menu is not visible, and with a swipe from right-to-left I can jump into the next post no problem. My problem is that in landscape mode the left side menu is enabled, and the swipe is not functioning properly anymore. Is there a possible fix for this?

**Comment by Alkarex** [comment]
Many things have changed in the upcoming version 1.13.0 (current dev). Could you please try the /dev branch with extensions disabled?

**Comment by pattems** [comment]
Swipe gestures seem to be provided by your TouchControl extension and (@Alkarex, correct me if I'm wrong) don't exist in the main project. Without looking at the code for the extension, it sounds like the swipe shortcuts are maybe only available when FreshRSS is in mobile view (less than 840px wide), and when you're in portrait mode it you're getting the desktop view, disabling the swipe gestures.

**Comment by aledeg** [comment]
@DOFfactory Is this still relevant?

**Comment by alfureu** [comment]
Yes still happening, the issue is not solved. Once you load FreshRSS webpage on your tablet, switch to landscape, and the frontend does not open any articles, you can just mark the articles read, but they actually do not open

**Comment by Alkarex** [comment]
@DOFfactory Could you please share a screenshot / screencast?

**Comment by alfureu** [comment]
Hi @Alkarex, not much to show, open the FreshRSS page article in normal view, turn the tabled (in my case Android, client: Bromite browser) by 90º and try clicking on the first article to read. It simply does not open (unfold), however, it is marked as "read".

**Comment by math-GH** [comment]
@DOFfactory  could you find the extension that makes the different between: it works fine and it creates the described issue?
